jprov/server/file_server.go

```diff
@@ -190,6 +192,7 @@ func StartFileServer(cmd *cobra.Command) {
 
 	ctx := utils.GetServerContextFromCmd(cmd)
 	ctx.Config.BaseConfig.LogLevel = logLevel
+	ctx.Logger = tmlog.NewTMLogger(tmlog.NewSyncWriter(os.Stdout)) // make new logger to consume the updated base config
```

Comment:
Upon initial testing--it appears this enables the jprovd command to take a custom tendermint-log-level on the very first run, but any future service restarts with a new 'tendermint-log-level' flag will not successfully set an updated log level. 

Perhaps a brand new *viper.Viper needs to be declared before line 194 aswell.